xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/displayers/displayer-text.html

```diff
@@ -17,28 +33,29 @@
  * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
  */
-
 define([
-    BASE_PATH + "displayers/defaultDisplayer.js",
-    "polyfills",
-  ], function (
-    DefaultDisplayer
-  ) {
+  "Vue",
+  BASE_PATH + "displayers/displayer-mixin.js",
+  "vue!" + BASE_PATH + "displayers/base-displayer.html",
+], function (
+  Vue,
+  displayerMixin
+) {
+
+  Vue.component("displayer-text", {
```

Comment:
Do we need both ``displayer-text`` and ``displayer-default``? They have the same code. On my side I've used "text" as default so far but I'm fine to change it to "default" (although "text" is a bit more explicit).

Reply:
Well, first I thought to used the default one even for text too, as the implementation is the same. But as the LD is meant to be reusable (by Cryptpad for instance), the other implementation could define a text filter / displayer that differs from the default one. And in that case, having a distinction between both is needed.

What we can also do is to put somewhere in the LD config which displayer and which filter should be used as the default one. Like that, we can set it to the text ones in our cases, and if needed, we could always change it easily to something else. WDYT?

Reply:
+1 to drop "default" displayer and to add ``meta.defaultDisplayer: 'text'`` (for our needs). Same for filters.